view/business-process-data-forms.js

```diff
@@ -28,10 +31,12 @@ exports.step = function () {
 		exports._forms.call(this)
 	);
 
+	nextStep = requirements._size > 0 ? 'documents' : costs.totalAmount > 0 ? 'pay' : 'submission';
+
 	insert(_if(and(eq(guideProgress, 1),
 		eq(dataForms._progress, 1)),
 		div({ class: 'user-next-step-button' },
-			a({ href: '/documents/' }, _("Continue to next step"))),
+			a({ href: '/' + nextStep + '/' }, _("Continue to next step"))),
```

Comment:
keeping the next step in a var is not reactive at all
use the _if and observable props of the progress everywhere pls